qiime2/sdk/util.py

```diff
@@ -92,3 +92,25 @@ def parse_format(format_str):
     except KeyError:
         raise TypeError("No format: %s" % format_str)
     return format_record.format
+
+
+def has_metadata(artifact):
```

Comment:
I'm in favoring of moving it now (see comment [here](https://github.com/qiime2/q2cli/pull/137#discussion_r115790323))